homeworks/Vladimir.Karbivnychiy_VladimirSkarb/8-components-templates/pictures-slider/src/app/interfaces/Image.ts

```diff
@@ -0,0 +1,18 @@
+export interface Image {
```

Comment:
I didn't mean merge all interfaces into the one)
I mean to just move all interfaces into one file. But this is not blocking